tasks/elasticsearch_activate.yml

```diff
@@ -47,14 +28,13 @@
     url: "https://{{ ansible_fqdn }}:9200/_xpack/license?acknowledge=true"
     body_format: json
     body: "{{ lookup('file', elasticsearch_host_license_file_path )  }}"
-    return_content: yes
-    user: "{{ elasticsearch_xpack_username | default('elastic') }}"
-    password: "{{ elasticsearch_xpack_password | default('changeme') }}"
-    validate_certs: no
+    return_content: true
+    # Using default user
+    user: elastic
```

Comment:
No chance that user can be something else? But I still want it to be variable anyway